storage/innobase/include/trx0trx.h

```diff
@@ -1152,17 +1159,23 @@ struct trx_t : ilist_node<>
     return false;
   }
 
-  /** @return logical modification time of a table only
-  if the table has bulk buffer exist in the transaction */
-  trx_mod_table_time_t *check_bulk_buffer(dict_table_t *table)
+  /**
+  @retval nullptr if the table doesn't have bulk buffer or
+  can skip sorting for primary key
+  @return logical modification time of a table only if the table
+  has bulk buffer exist */
```

Comment:
The `@return` comment usually is before `@retval`. The "if the table has…" part in it seems to have been made redundant by the `@retval nullptr` comment.